PKGBUILD

```diff
@@ -2,44 +2,43 @@
 # Contributor: Philipp Claßen <philipp.classen@posteo.de>
 
 pkgname=lib32-benchmark
-pkgver=1.5.0
+pkgver=1.6.0
 pkgrel=1
-pkgdesc="A microbenchmark support library, by Google (32-bit)"
+pkgdesc="A microbenchmark support library (32-bit)"
 arch=('x86_64')
 url="https://github.com/google/benchmark"
 license=('Apache')
 depends=('lib32-gcc-libs')
 makedepends=('cmake' 'gcc-multilib')
 
-source=("https://github.com/google/benchmark/archive/v${pkgver}.tar.gz")
-sha256sums=('3c6a165b6ecc948967a1ead710d4a181d7b0fbcaa183ef7ea84604994966221a')
+source=("https://github.com/google/benchmark/archive/v$pkgver/benchmark-$pkgver.tar.gz")
+sha256sums=('1f71c72ce08d2c1310011ea6436b31e39ccab8c2db94186d26657d41747c85d6')
 
 prepare() {
-  cd "${srcdir}/benchmark-${pkgver}"
-
-  mkdir -p build && cd build
-
-  export CFLAGS="-m32 ${CFLAGS}"
-  export CXXFLAGS="-m32 ${CXXFLAGS}"
-  export LDFLAGS="-m32 ${LDFLAGS}"
-  export PKG_CONFIG_PATH='/usr/lib32/pkgconfig'
-
-  cmake .. -DCMAKE_BUILD_TYPE="Release" \
-           -DCMAKE_INSTALL_PREFIX=/usr \
-           -DCMAKE_INSTALL_LIBDIR=lib32 \
-           -DBUILD_SHARED_LIBS=ON \
-           -DBENCHMARK_ENABLE_LTO=ON \
-           -DBENCHMARK_ENABLE_INSTALL=ON \
-           -DBENCHMARK_ENABLE_GTEST_TESTS=OFF
+  cd "$srcdir/benchmark-$pkgver"
+  mkdir -p build
 }
 
 build() {
-  cd "${srcdir}/benchmark-${pkgver}/build"
+  cd "$srcdir/benchmark-$pkgver/build"
+
+  cmake \
+    -DCMAKE_BUILD_TYPE=None \
```

Comment:
Isn't it supposed to be release or a similar one?

Reply:
I thought so to, but this is what the 64-bit Benchmark package is doing. I'm not sure why, though.

https://github.com/archlinux/svntogit-community/blob/packages/benchmark/trunk/PKGBUILD#L26

Reply:
I've collected some information and I believe this is a bug in packaging. I've filed a bug: https://bugs.archlinux.org/task/73185 and notified maintainers. Let's wait until they respond.